llama_index/query_engine/retriever_query_engine.py

```diff
@@ -165,7 +165,7 @@ async def _aquery(self, query_bundle: QueryBundle) -> RESPONSE_TYPE:
         )
 
         retrieve_id = self.callback_manager.on_event_start(CBEventType.RETRIEVE)
-        nodes = self._retriever.retrieve(query_bundle)
+        nodes = await self._retriever.aretrieve(query_bundle)
```

Comment:
with recent changes, this will actually skip calling node-post processors, since node post-processors are applied in `self.retrieve`

Reply:
I fixed this :)